File: Project/Backend/daariiou_photography_backend/daariiou_photography_backend/DTO/UserDTO.cs
using System.Collections.Generic;

namespace daariiou_photography_backend.DTO
{
    public class UserDTO
    {
        public int Uid { get; set; }
        public string Username { get; set; }
        public string Name { get; set; }
        public string Lastname { get; set; }
        public string Password { get; set; }
        public string IsAdmin { get; set; }
        public List<PictureDTO> Pictures { get; set; }
        public List<ShootingDTO> Shootings { get; set; }
    }
}

File: Project/Frontend/daariiou-photography-frontend/src/api/lib/models/user-dto.ts
/* tslint:disable */
/* eslint-disable */
import { PictureDto } from './picture-dto';
import { ShootingDto } from './shooting-dto';
export interface UserDto {
  isAdmin?: null | string;
  lastname?: null | string;
  name?: null | string;
  password?: null | string;
  pictures?: null | Array<PictureDto>;
  shootings?: null | Array<ShootingDto>;
  uid?: number;
  username?: null | string;
}
